pallets/order-book/src/lib.rs

```diff
@@ -453,7 +460,17 @@ pub mod pallet {
             min_lot_size: OrderVolume,
             max_lot_size: OrderVolume,
         ) -> DispatchResult {
-            ensure_root(origin)?;
+            let origin_check_result = T::ParameterUpdateOrigin::ensure_origin(origin)?;
+            match origin_check_result {
+                Either::Left(who) => {
+                    ensure!(
+                        T::AssetInfoProvider::is_asset_owner(&order_book_id.base, &who)
+                            || T::AssetInfoProvider::is_asset_owner(&order_book_id.quote, &who),
```

Comment:
Remove. The owner of `base` asset can update order book, not the owner of `quote` asset

Reply:
https://github.com/sora-xor/sora2-network/issues/400#issuecomment-1571702153